guides/common/modules/proc_upgrading-project-or-proxy-in-place-using-leapp.adoc

```diff
@@ -13,12 +13,25 @@ ifdef::foreman-el,foreman-deb,katello[]
 For more information, see {ReleaseNotesDocURL}[Release notes].
 endif::[]
 * {Project} {ProjectVersion} or {SmartProxy} {ProjectVersion} running on {EL} 8.
+ifdef::satellite[]
+* For {SmartProxy} upgrade, enable and synchronize the following repositories to {ProjectServer}:
++
+Red Hat Enterprise Linux 9 for x86_64 - AppStream (RPMs):: `rhel-9-for-x86_64-appstream-rpms x86_64`
++
+Red Hat Enterprise Linux 9 for x86_64 - BaseOS (RPMs):: `rhel-9-for-x86_64-baseos-rpms x86_64`
++
+Red Hat Satellite Capsule 6.16 for RHEL 9 x86_64 RPMs:: `satellite-capsule-6.16-for-rhel-9-x86_64-rpms`
```

Comment:
I suggest to use an attribute in favor of "6.16": https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-satellite.adoc#L3